### DISCUSSION TITLE
No documentation on how to force handle to be focused

### DISCUSSION THREAD

**Discussion by henryjameslau** [simeydotme/svelte-range-slider-pips#154] [Formatting / Styling] [answered]
**Describe the feature**
No documentation on how to force handle to be focused

**Explain it's value / reasoning**
I think this would be useful. It's used in some of the documentation.

**Comment by simeydotme** [comment]
just want to highlight the fact you wrote;
> no documentation 

and then

> it's [used] in [some of] the documentation 

😂 

thanks though, I get you wanted to know explicitly how to set it. 👍

**Comment by henryjameslau** [comment]
Ha ha, sorry. That was poorly written.

**Comment by zuckerruebe** [comment]
I'm also looking to always have the handle show, i.e. I'd like the user to see the selected value always.

**Comment by simeydotme** [comment]
> I'm also looking to always have the handle show, i.e. I'd like the user to see the selected value always.

If you just want to _ALWAYS_ show labels/handles; that's CSS :)

```svelte
<RangeSlider id="always" float />

<style>
:global(#always .rangeHandle .rangeFloat) {
    opacity: 1;
    top: -.2em;
    transform: translate(-50%, -100%);
}
</style>
```

^ this should work, but I can't show you in a REPL/Demo right now as Svelte team launched version 5 and the whole Svelte Playground is broken right now

**Comment by simeydotme** [comment]
Here's a working "always show handle" slider

https://svelte.dev/playground/030797781fd64ad88302d1343f5b2c43?version=5.1.0#H4sIAAAAAAAAE22PQWvDMAyF_4rQKLRQt-lgF6ct7DJ23o51D26iBINjG1vtGkL--3Cysh12k773nsQbsDGWEsrTgNwHQjkBXKPTXd5eQ9ikG1nO7KIT_ccr75gcJ5S4T1U0gY_KKTZd8JHhQ7uWPq2pKUITfQcK56SIWRFpkkQwISksc9ASw03bKyU4wGlXnEvl9tvfy27_96apDwq1_dJ9UgiN9ZphO9sS95aOysnW-ou2y6fZBpvp87t2taWf5S3nVjAoBwDgg64M9xJ25QzYBwli80zdA0TtUuNjJ-fRaqaleCkWaxC7olisSuXGXObx--LrfgUDZKY46Lo2rpXwUoR7OTOmOwttTeskVOSY4iSMU_mpCa4xm1ByvNJ4Hr8Be6pI6L8BAAA=

**Comment by simeydotme** [comment]
> **Describe the feature** No documentation on how to force handle to be focused
> 
> **Explain it's value / reasoning** I think this would be useful. It's used in some of the documentation.

as for focussing on load;
https://svelte.dev/playground/030797781fd64ad88302d1343f5b2c43?version=5.1.0

Here's a demo, I'll try to add it to Docs soon :)

**Comment by simeydotme** [comment, marked as answer]
https://simeydotme.github.io/svelte-range-slider-pips/en/examples/float/#focus-on-load
^ here we go :)

**Comment by zuckerruebe** [comment]
Great stuff, thanks!
